crates/context_aware_config/src/api/dimension/handlers.rs

```diff
@@ -119,10 +120,22 @@ async fn create(
 async fn get(
     db_conn: DbConnection,
     tenant_config: TenantConfig,
+    filters: Query<QueryFilters>,
 ) -> superposition::Result<HttpResponse> {
     let DbConnection(mut conn) = db_conn;
 
-    let result: Vec<Dimension> = dimensions.get_results(&mut conn)?;
+    let n_dimensions: i64 = dimensions.count().get_result(&mut conn)?;
+    let mut builder = dimensions.into_boxed().order(created_at.desc());
+    if let Some(limit) = filters.count {
+        builder = builder.limit(limit);
+    }
+    if let Some(page) = filters.page {
+        let offset = (page - 1) * filters.count.unwrap_or(10);
+        builder = builder.offset(offset);
+    }
+    let limit = filters.count.unwrap_or(10);
+    let result: Vec<Dimension> = builder.load(&mut conn)?;
```

Comment:
Could we apply the same API behaviour here, where if limit and page are not passed they are assumed?

Reply:
@Datron the default-config page with the grouping enabled needs the whole default-config.